lib/nav/sort-selector.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="sort-selector" :class="{ 'is-vertical': vertical }">
+    <ui-radio class="sort-selector-radio" color="accent" label="Recently Updated" :value="selectedSort" trueValue="updateTime" @input="input" />
+    <ui-radio class="sort-selector-radio" color="accent" label="Recently Published" :value="selectedSort" trueValue="publishTime" @input="input" />
+    <ui-radio class="sort-selector-radio" color="accent" label="Recently Scheduled" :value="selectedSort" trueValue="scheduledTime" @input="input" />
+  </div>
+</template>
+
+<script>
+  import UiRadio from 'keen/UiRadio';
+
+  export default {
+    props: ['selectedSort', 'vertical'],
+    methods: {
+      input(sortField) {
+        this.$emit('select', sortField);
+      }
+    },
+    components: {
+      UiRadio
+    }
+  };
+</script>
+
+<style lang="sass">
```

Comment:
could you put the styles at the beginning? I see this convention in others components